iiwa_driver/src/iiwa.cpp

```diff
@@ -36,6 +36,33 @@
 #include <thread>
 
 namespace iiwa_ros {
+    bool hasRealtimeKernel() {
```

Comment:
Can you make those two global functions in snake case (i.e. with lower case and underscores?) to follow the rest of the styling of the code? Thanks.. Also I believe that those functions need to be inline for better compiler optimization...